sigstore/_internal/timestamp.py

```diff
@@ -20,6 +20,17 @@
 from dataclasses import dataclass
 from datetime import datetime
 
+import requests
+from rfc3161_client import (
+    TimestampRequestBuilder,
+    TimeStampResponse,
+    decode_timestamp_response,
+)
+
+from sigstore._internal import USER_AGENT
+
+CLIENT_TIMEOUT: int = 30
```

Comment:
This seems like a really long timeout. How was this picked, and can we just use the default timeout instead?

Reply:
I used `sigstore-go` [value](https://github.com/sigstore/sigstore-go/blob/e92142f0734064ebf6001f188b7330a1212245fe/pkg/sign/timestamping.go#L57)